compile.sh

```diff
@@ -31,6 +31,12 @@ FORCE_CHECKOUT="yes"			# ignore manual changes to source
 BUILD_ALL="no"				# cycle through available boards and make images or kernel/u-boot packages.
 					# set KERNEL_ONLY to "yes" or "no" to build all packages/all images
 
+PRIVATE_CCACHE="no"			# Use separate ccache directory located in build tree ($DEST/ccache)
+					# Can be used to avoid problems with ownershipt on $HOME/.ccache. 
+					# Set to "yes" if you are using build script as non-root user with "sudo"
+					# note: to use a ramdisk for ccache, create first a symlink to $DEST/ccache
+                                        # Set to "no" check if $DEST/ccache exists and delete (directory/symlink)
```

Comment:
I'm not sure I understand. It will automagically delete $dest/ccache and then use $home/.ccache ? I'll let the user do the cleanup himself, eventually printing a message that "there's a trailing ccache directory at $dest/ccache" because he may still want to use it later or he made the  mistake to change the variable, or whatever. The content may be worth hours of compilation...